### ISSUE TITLE
Blizzard support

### ISSUE THREAD

**Issue by snormonster** [jamie-mh/AuthenticatorPro#54]
Hi thanks for a great app, does anyone know if there's any way to use Authenticator Pro to generate codes for Blizzard / Battlenet? I don't want to use their app just for this.

I wasn't able to figure out if it is possible or not, but it seems that some sort of Serial key needs to be generated in the auth app. I found some info at this link which shows how other third party auth apps can be used for generating TOTP codes https://kangaroobyte.com/blizzard-authenticator-using-andotp-freeotp-and-other-totp-apps-20200403-227/

**Comment by jamie-mh** [comment]
Hi @snormonster ,

Indeed, you will need to follow the guide that you linked. Since Blizzard's authenticator is proprietary, this is the only way of converting it to a standard TOTP. If you generate a QR code with this method and scan it in Authenticator Pro, it will work.

**Comment by snormonster** [comment]
I got a bit lost on that guide, so I followed the "Requesting a new authenticator" bit on https://github.com/jleclanche/python-bna and I generated a Serial + Secret.

This I then entered into Authenticator Pro manually, I typed Blizzard as the provider, left out username, left it on TOTP using SHA1, but I changed the digits to 8 with 30s period. It created an entry (with the Blizzard icon no less) and it seems to work with my Blizzard account. I hope this method is safe.
